client/components/embed-container/index.jsx

```diff
@@ -148,6 +151,40 @@ function embedTumblr( domNode ) {
 	}, 30 );
 }
 
+let smallipopLoader;
+function embedSmallipop( domNode ) {
+	debug( 'processing smallipop for', domNode );
+	if ( smallipopLoader ) {
+		return;
+	}
+
+	// TODO - put the CSS/JS files in a better location
+	loadjQueryDependentScript( hovercard_URL_Prefix + 'jquery.smallipop.js', function() {
+		smallipopLoader = true;
```

Comment:
this set should be outside the load callback to prevent more than one attempt at loading